buildbot_gitea/reporter.py

```diff
@@ -157,12 +157,13 @@ def send(self, build):
                 )
                 if res.code not in (200, 201, 204):
                     message = yield res.json()
-                    message = message.get('message', 'unspecified error')
+                    message = message[0].get('message', 'unspecified error')
```

Comment:
Do you get an array here? My unittests which uses actual intercepted data from a gitea response to verify this. I need to check if the message return value is an array now.

Reply:
Yes, I got an array, but sadly I can't remember for what error it was. I'm using Gitea version 8d40032.